data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/S3ObjectWorker.java

```diff
@@ -52,9 +52,11 @@ public S3ObjectWorker(final S3Client s3Client,
     }
 
     void parseS3Object(final S3ObjectReference s3ObjectPointer) throws IOException {
```

Comment:
can you update the variable naming here as well? We are using pointer and reference interchangeably here and it would be nice to consistent and stick with reference